toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -371,11 +378,13 @@ texinfo-6.5-7.cm1.aarch64.rpm
 texinfo-debuginfo-6.5-7.cm1.aarch64.rpm
 unzip-6.0-16.cm1.aarch64.rpm
 unzip-debuginfo-6.0-16.cm1.aarch64.rpm
-util-linux-2.32.1-3.cm1.aarch64.rpm
-util-linux-debuginfo-2.32.1-3.cm1.aarch64.rpm
-util-linux-devel-2.32.1-3.cm1.aarch64.rpm
-util-linux-lang-2.32.1-3.cm1.aarch64.rpm
-util-linux-libs-2.32.1-3.cm1.aarch64.rpm
+unzip-6.0-15.cm1.aarch64.rpm
+unzip-debuginfo-6.0-15.cm1.aarch64.rpm
+util-linux-2.32.1-4.cm1.aarch64.rpm
+util-linux-debuginfo-2.32.1-4.cm1.aarch64.rpm
+util-linux-devel-2.32.1-4.cm1.aarch64.rpm
+util-linux-lang-2.32.1-4.cm1.aarch64.rpm
+util-linux-libs-2.32.1-4.cm1.aarch64.rpm
```

Comment:
It looks like you've removed `util-linux` from being built as part of the toolchain, so we should remove it here as well. Otherwise builds will start failing, since that package will no longer be available in the folder with built toolchain RPMs.